### PR TITLE
Alias onboarding analytics

### PR DESCRIPTION
**Pull request by mplorentz**
#1135

### CODE REVIEW

File: Source/UI/RoomListView.swift [hunk 1]
```diff
@@ -131,14 +138,32 @@ struct RoomListView<ViewModel>: View where ViewModel: RoomListViewModel {
...
             }
+            
+            if viewModel.showJoinPlanetaryRoomButton {
+                Section {
+                    Button("Join Planetary Room") {
+                        viewModel.joinPlanetaryRoom()
+                    }
+                    .foregroundColor(.primaryAction)
+                    Text(
+                        "Joining the official Planetary room server will allow to register aliases like " +
```
Comment (martindsq) [inline Source/UI/RoomListView.swift:149]: This string needs to be localized

File: Source/UI/RoomListView.swift [hunk 2]
```diff
@@ -131,14 +138,32 @@ struct RoomListView<ViewModel>: View where ViewModel: RoomListViewModel {
                     .font(.subheadline)
                     .padding(.top, 4)
             }
+            
+            if viewModel.showJoinPlanetaryRoomButton {
+                Section {
+                    Button("Join Planetary Room") {
```
Comment (martindsq) [inline Source/UI/RoomListView.swift:144]: This string needs to be localized

File: Source/UI/RoomListView.swift [hunk 3]
```diff
@@ -183,15 +212,17 @@ fileprivate class PreviewViewModel: RoomListViewModel {
                 self.loadingMessage = nil
             }
         } else {
-            errorMessage = "Error joining room"
+            alertMessage = "Error joining room"
```
Comment (martindsq) [inline Source/UI/RoomListView.swift:215]: This string needs to be localized

**Review by martindsq — CHANGES_REQUESTED** [review]

**Comment by martindsq** [comment]
Just a couple of strings that need to be localized and a CHANGELOG entry is needed here to close the PR.

As a side note: I've seen that you don't use this https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue in your PR descriptions, I find them very handy to automatically link issues when opening a PR.

**Comment by mplorentz** [comment]
> As a side note: I've seen that you don't use this https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue in your PR descriptions, I find them very handy to automatically link issues when opening a PR.

I probably should. I usually go back and read the ticket and comments again after I click "merge" to make sure it is ready to be closed, so I'm not thinking about closing it when I'm creating the PR. But it would probably be more clear to do it that way.

**Pull request merged**